diode-server/docker/scripts/dev-auth-manage.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+join_with_spaces() {
+  local IFS=" "
+  echo "$*"
+}
+
+script_path=$(dirname "$0")
+cd ${script_path}/../..
+make auth_manage_command="$(join_with_spaces "$@")" docker-compose-dev-auth-manage
```

Comment:
this is a little script for local development use since it's awkward to pass flags to a make target... not strictly needed